modules/model/src/main/kotlin/jp/co/soramitsu/iroha2/generated/ConfigurationEvent.kt

```diff
@@ -53,7 +53,7 @@ public sealed class ConfigurationEvent : ModelEnum {
      * 'Created' variant
```

Comment:
we don't edit `generated` module entities btw

Reply:
ok, that makes sense

Reply:
we make changes in `codegen`

Reply:
fixed, check it now